src/test/java/org/mockitousage/matchers/CapturingArgumentsTest.java

```diff
@@ -298,7 +313,7 @@ public void captures_correctly_when_captor_used_multiple_times() throws Exceptio
 
         // then
         // this is only for backwards compatibility. It does not make sense in real to do so.
-        verify(mock).mixedVarargs(any(), argumentCaptor.capture(), argumentCaptor.capture(), argumentCaptor.capture());
+        verify(mock).mixedVarargs(any(), argumentCaptor.capture());
```

Comment:
@terebesirobert Hi, I actually fail to see what was changed. Do you mind not squashing the commits next time? 

>  If it's already added, we shouldn't add it. What do you think?

That may break somebodys test. Some may rely on the fact that some instances were passed multiple times.

If you like you can tryout this attempt to fix the issue,i sadly can't test it:
```
    private void captureVarargsPart(Invocation invocation) {
        if (!invocation.getMethod().isVarArgs()) {
            return;
        }

        Object[] rawArguments = invocation.getRawArguments();
        int indexOfVararg = rawArguments.length - 1;

        Object varargArray = rawArguments[indexOfVararg];
        int size = Array.getLength(varargArray);

        for (int i = 0; i < size; i++) {
            ArgumentMatcher m = matchers.get(i+indexOfVararg);
            if (m instanceof CapturesArguments) {
                ((CapturesArguments) m).captureFrom(Array.get(varargArray, i));
            }
        }
    }
```

Hope it is self describing, I am currently short at time.